package.json

```diff
@@ -26,7 +26,8 @@
 			"selector-type-no-unknown": null,
 			"declaration-no-important": null,
 			"selector-class-pattern": null,
-			"selector-id-pattern": null
+			"selector-id-pattern": null,
+			"selector-max-universal": null
```

Comment:
@sindresorhus this rule seems rather arbitrary

Reply:
`*` is very inefficient and far-reaching. I think it makes sense to limit it in normal code. RG is an expectation since it has to do a lot of overrides, that you wouldn't do in a normal codebase.